### ISSUE TITLE
sage.{misc,monoids,sets}: Replace imports from sage.*.all for namespace packages

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage#34956]
<div id="comment:0"></div>

Using `./sage -fiximports` from #34945.

Also remove trailing whitespace in the affected files.

Part of Meta-ticket #34201

CC:  @alexchandler100 @tscrim

Component: **refactoring**

Author: **Alex Chandler, Matthias Koeppe**

Branch/Commit: **[u/mkoeppe/sage__misc_monoids_sets___replace_imports_from_sage___all_for_namespace_packages](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/sage__misc_monoids_sets___replace_imports_from_sage___all_for_namespace_packages) @ [`a8a3d6e`](https://github.com/sagemath/sagetrac-mirror/commit/a8a3d6eb0a9c4e9d079424c234c7f4abaa9d288f)**

Reviewer: **Travis Scrimshaw**

_Issue created by migration from https://trac.sagemath.org/ticket/34956_

**Comment by mkoeppe** [comment]
Branch: **[u/mkoeppe/sage__misc_monoids_sets___replace_imports_from_sage___all_for_namespace_packages](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/sage__misc_monoids_sets___replace_imports_from_sage___all_for_namespace_packages)**

**Comment by mkoeppe** [comment]
Commit: **[`ecdec6b`](https://github.com/sagemath/sagetrac-mirror/commit/ecdec6bc53f136282428b64516c4b436304af9e5)**

**Comment by mkoeppe** [comment]
<div id="comment:2"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/b56aaae80ce5139127e6b9c7ea787440872bd963"><code>b56aaae</code></a></td><td><code>sage -fiximports src/sage/{misc,monoids,sets}</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/ecdec6bc53f136282428b64516c4b436304af9e5"><code>ecdec6b</code></a></td><td><code>src/sage/misc/lazy_import.pyx: Do not use .all imports as examples in doctests</code></td></tr></table>

**Comment by mkoeppe** [comment]
Changed author from **Alex Chandler** to **Alex Chandler, Matthias Koeppe**

**Comment by mkoeppe** [comment]
<div id="comment:3" align="right">comment:3</div>


```
sage -t --random-seed=177204116540218245270635058655148957979 sage/misc/lazy_import.pyx  # 12 doctests failed
```

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:4"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/d1ca407227f0bd17414be8a3a30ba131d855f0e4"><code>d1ca407</code></a></td><td><code>src/sage/misc/lazy_import.pyx: Fix doctest</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/af823ccee98d7b0a4e21dec01fbd08e7753e8c1c"><code>af823cc</code></a></td><td><code>src/sage/misc/lazy_import.pyx: Fix doctests</code></td></tr></table>

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`ecdec6b`](https://github.com/sagemath/sagetrac-mirror/commit/ecdec6bc53f136282428b64516c4b436304af9e5)** to **[`af823cc`](https://github.com/sagemath/sagetrac-mirror/commit/af823ccee98d7b0a4e21dec01fbd08e7753e8c1c)**

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`af823cc`](https://github.com/sagemath/sagetrac-mirror/commit/af823ccee98d7b0a4e21dec01fbd08e7753e8c1c)** to **[`16efed0`](https://github.com/sagemath/sagetrac-mirror/commit/16efed081a7ddcee87f759ceadea8d587844d847)**

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:5"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/16efed081a7ddcee87f759ceadea8d587844d847"><code>16efed0</code></a></td><td><code>src/sage/misc/sage_eval.py: Fix markup</code></td></tr></table>

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`16efed0`](https://github.com/sagemath/sagetrac-mirror/commit/16efed081a7ddcee87f759ceadea8d587844d847)** to **[`a8a3d6e`](https://github.com/sagemath/sagetrac-mirror/commit/a8a3d6eb0a9c4e9d079424c234c7f4abaa9d288f)**

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:7"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/39b7ca4f98f2b706cc531401a3445fa6b5b4c8ca"><code>39b7ca4</code></a></td><td><code>src/sage/env.py: Hide a legitimate .all import from relint</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/a8a3d6eb0a9c4e9d079424c234c7f4abaa9d288f"><code>a8a3d6e</code></a></td><td><code>src/sage/tests/startup.py: Hide a legitimate .all import from relint</code></td></tr></table>

**Comment by tscrim** [comment]
<div id="comment:9" align="right">comment:9</div>

LGTM.

**Comment by mkoeppe** [comment]
<div id="comment:11" align="right">comment:11</div>

Thanks!

**Comment by mkoeppe** [comment]
Reviewer: **Travis Scrimshaw**

**Comment by mkoeppe** [comment]
Merged in 10.0.beta0
